contrib/screener-api/db/models.go

```diff
@@ -93,3 +145,7 @@ func MakeRecord(address string, records []trmlabs.AddressRiskIndicator) *Address
 var _ schema.GormDataTypeInterface = addressRiskIndicators{}
 var _ driver.Value = addressRiskIndicators{}
 var _ sql.Scanner = &addressRiskIndicators{}
+
+var _ schema.GormDataTypeInterface = BlacklistedAddress{}
```

Comment:
Also not needed (See above comment). I think you may be confusing `addressRiskIndicators` (a data type) with `AddressIndicators` a table

Reply:
yes i was haha makes more sense now